src/patternfly/components/Label/label-editable-text.hbs

```diff
@@ -0,0 +1,15 @@
+<{{#if label-editable-text--type}}{{label-editable-text--type}}{{else if label--IsEditableActive}}input{{else}}button{{/if}} class="pf-c-label__editable-text{{#if label-editable-text--modifier}} {{label-editable-text--modifier}}{{/if}}"
+  id="{{label--id}}-editable-text"{{#if label--IsEditableActive}} type="text" value="{{~> @partial-block}}"{{else}} currvalue="{{~> @partial-block}}"{{/if}}
```

Comment:
Is there a reason this isn't multi-line?

```suggestion
  id="{{label--id}}-editable-text"
  {{#if label--IsEditableActive}}
    type="text"
    value="{{~> @partial-block}}"
  {{else}}
    currvalue="{{~> @partial-block}}"
  {{/if}}
```

Reply:
just an artifact. I'll fix it.